contracts/core/zapper/WidoZapperGammaAlgebra.sol

```diff
@@ -165,7 +165,7 @@ contract WidoZapperGammaAlgebra is WidoZapper_ERC20_ERC20 {
         bytes32 positionKey = keccak256(abi.encodePacked(address(hyper), tickLower, tickUpper));
         (uint position, , , ,) = pool.positions(positionKey);
         uint128 liquidity = uint128(position.mul(shares) / hyper.totalSupply());
-        (uint160 sqrtRatioX96, , , , , ,) = pool.globalState();
+        uint160 sqrtRatioX96 = _sqrtRatioX96(address(pool));
```

Comment:
can we pass this in from _getAmountsOut so that it doesn't have to be fetched multiples times?

Reply:
Hard to get it right, when moving things out there's a stack too deep